versioned_docs/version-1.3.0/release-notes/release-notes-for-kubeslice-oss-1.3.1.mdx

```diff
@@ -0,0 +1,19 @@
+# Release Notes for KubeSlice OSS 1.3.1
+
+*Release date: 26th Apr 2024*
+
+KubeSlice is a ***cloud-independent*** platform that combines network, application, Kubernetes, and deployment services 
+in a framework to accelerate application deployment in a multi-cluster and multi-tenant environment. KubeSlice achieves this 
+by creating logical application ***slice*** boundaries that enable seamless communication between pods and services across 
+clusters, clouds, edges, and data centers.
+
+We continue to add new features and enhancements to KubeSlice. 
+
+## What's New :speaker:
+
+Installing KubeSlice in the Air-Gapped Environment
```

Comment:
KubeSlice already supports Air-gapped installation  - no? 

So, emphasize usage of kubeslice-cli in the heading. 

`Installing KubeSlice in and Air-Gapped Environment using kubeslice-cli` or something of that sort ...

Reply:
done